packages/stg-evm-v2/devtools/config/testnet/usdt-token.config.ts

```diff
@@ -41,6 +50,13 @@ export default async (): Promise<OmniGraphHardhat<OwnableNodeConfig, unknown>> =
             //         owner: bl3AssetAddresses.USDT,
             //     },
             // },
+            // TODO comment out for github ci/cd?
+            {
+                contract: odysseyUSDT,
+                config: {
+                    owner: odysseyAssetAddresses.USDT,
+                },
+            },
```

Comment:
does this need to be commented out?

Reply:
This builds are going to fail regardless due to a issue further along as well, but yes nice catch i will comment this out since it is already configured